documents/Strategies/Destruction strategy.tex

```diff
@@ -0,0 +1,44 @@
+\documentclass{article}
+
+\usepackage{graphicx}
+
+\author{Job Zoon}
+\title{Strategy for destroying buildings}
+
+\begin{document}
+\pagenumbering{gobble}
+
+\maketitle{}
+\newpage{}
+
+\section{Introduction}
+In this document the strategy about destroying buildings will be explained. In the first section the reason to destroy a building will be defined, and in the second section the choice for a particular building will be clarified. This document will be used to implement the following user story:\\
+\\
+\textit{"As an agent\\
+When in a simulation\\
+I am able to destroy stuff if I think 
+this is benificial for me\\
+To make room for construction or to 
+optimize a certain indicator."\\}
+
+There are two important parts in this user story that need to be discussed before implementing anything. The 'stuff' part: so which buildings need to be destroyed, and the 'benificial' part: so when buildings need to be destroyed. Again, these two questions will be discussed in the following two sections. 
+
+\section{When/why to destroy buildings}
+In this section the reasons to destroy a building will be explained. There are two types of reasons and both of them ultimately give a better indicator score. The first reason is to directly influence an indicator, and the second reason is to make space for a new building that influences an indicator. Both will be discussed in the following sections. The agent needs to keep destroying buildings in order to build newer, higher buildings, until all indicators are at 100\%.
```

Comment:
Maybe remove the sentence "Both will be discussed in the following sections." as this is stating exactly the same things you said at the start of the paragraph.